### ISSUE TITLE
Icarus Server Not Listening on Port 17777

### ISSUE THREAD

**Issue by wizzbangwa** [patrix87/PowerShellGSM#33]
Followed your instructions.  Server is finally running after suffering through figuring out how to get rid of the Visual C++ error message.  I am on a fresh Windows 11 install.  The Icarus server is not visible in the "Dedicated Servers" list, and is not available by direct connect either.

- netstat -an | Select-String 27015 shows the server machine is listening on the port.
- netstat -an | Select-String 17777 doesn't return anything, showing the server is not listening on this port.
- icarus.psm1 is set to both ports above.
- I have opened ports 27015 and 17777 TCP and UDP on both routers (I have two separate networks).
- Windows firewall has all ports opened for Icarus for UDP and TCP for private, public, and domain networks.
- The server's console window is open and showing the server is running, but no confirmation the server is listening on what ports.
- The server does not show in the "Dedicated Servers" list.
- The server is not available through direct connect in Icarus.

Don't know what to do from here.

**Comment by patrix87** [comment]
Can you post your configuration and logs ?

**Comment by wizzbangwa** [comment]
I managed to get it running a few minutes ago. Shortly after my original post I switched to my own startup cmd which got everything running.  This time around I tried the automated script by [@BananaAcid](https://github.com/BananaAcid/).  Now the server is running just fine.  Now I need to configure daily or hourly automated backups, if that's possible.  So far I have to manually  restart the launcher script, not quite helpful if someone wipes out the prospect world and I haven't restarted recently.  Happened once already.
